hw_dialog06_scale/controllers/dialog06_protocol.py

```diff
@@ -35,31 +37,40 @@ def _parse_checksums_request(data):
     d0, z = None, None
 
     record_no = _parse_record_no(data)
-    data = data.decode('utf-8')
-    if record_no == u'11':
-        esc = data.index(u'\x1b')
+    data = data.decode("utf-8")
+    if record_no == "11":
+        esc = data.index("\x1b")
         d0 = data[esc + 1]
-        if d0 == u'2':
-            z = data[data.index(d0) + 1:data.index(d0) + 3]
-    _logger.debug('[PARSE][CHECKSUMS] d0 : {} and z : {}'.format(d0, z))
+        if d0 == "2":
+            z = data[data.index(d0) + 1 : data.index(d0) + 3]
+    _logger.debug("[PARSE][CHECKSUMS] d0 : {} and z : {}".format(d0, z))
     return d0, z
 
 
 def _parse_weighing_result(data):
     """ Parse Record 02, returning unit of measure (uom), weight, unit price and total price values """
-    record_no, uom, scale_weight, unit_price, total_price = None, None, None, None, None
+    record_no, uom, scale_weight, unit_price, total_price = (
+        None,
+        None,
+        None,
+        None,
+        None,
+    )
 
     record_no = _parse_record_no(data)
-    data = data.decode('utf-8')
-    if record_no == u'02':
-        esc_indices = [i for i, x in enumerate(data) if x == u'\x1b']
+    data = data.decode("utf-8")
+    if record_no == "02":
```

Comment:
tu remplaces partout u'xx' par "xx". c'est ok de virer le u ? Je suppose que c'est parce que c'est python 3

Reply:
C’est ok : 
« In Python 3.x the strings use Unicode by default and there's no need for the u prefix »
src : https://stackoverflow.com/questions/2464959/whats-the-u-prefix-in-a-python-string

PS : c’est le formater black qui s’en est chargé